src/app/shared/components/star-rating/star-rating.component.ts

```diff
@@ -37,7 +37,6 @@ export class StarRatingComponent implements OnInit, ControlValueAccessor {
   onTouched = () => { };
 
   writeValue(quantity: number) {
-    this.rate = quantity;
```

Comment:
Why is it removed

Reply:
this function calling it self and the star rating is adding automatically